### PR TITLE
[*] regression script maintenance to ensure stability

### PR DESCRIPTION
**Pull request by Alex-Grubba-CDS**

### CODE REVIEW

File: futurenhs.automation/features/search.feature [hunk 1]
```diff
@@ -47,14 +46,13 @@ Feature: Search
 
 
     Scenario: FNHS100 - Search for a Group That Doesn't Exist
-        When I search for 'Automation Group'
+        When I search for 'Automaton Group'
```
Comment (NathanGoodall2019) [inline futurenhs.automation/features/search.feature:49]: This is spelt wrong you sure this won't effect your run in correlation to the data?

File: futurenhs.automation/features/search.feature [hunk 2]
```diff
@@ -133,7 +125,6 @@ Feature: Search
 
     Scenario: FNHS108 - Search for a Group By the Strapline
         When I search for 'DO NOT USE - This group is reserved solely for use by our automated test scripts'
-        Then there are '6' search results displayed
```
Comment (NathanGoodall2019) [inline futurenhs.automation/features/search.feature:136]: I presume you have decided to remove the result count due to instability on data?

File: futurenhs.automation/features/userAccess.feature [hunk 3]
```diff
@@ -1,111 +1,112 @@
 Feature: User Access
     User journeys around users accessing the portal, including login page, registration, and unauthenticated page restrictions.
 
-Background:
-    Given I have navigated to '/'
+    Background:
+        Given I have navigated to '/'
```
Comment (NathanGoodall2019) [inline futurenhs.automation/features/userAccess.feature:5]: Is there a particular reason as to why you are doing a lot of formatting changes?

File: futurenhs.automation/wdio.conf.js [hunk 4]
```diff
@@ -41,9 +41,9 @@ exports.config = {
     ],
     suites: {
         fullRegression: [
-            './features/**/platformAdmin.feature',
             './features/**/userAccess.feature',
             './features/**/search.feature',
+            './features/**/platformAdmin.feature',
```
Comment (NathanGoodall2019) [inline futurenhs.automation/wdio.conf.js:46]: Will be useful at some stage to do a further run on features to see which run fast and to organise them accordingly when possible.

**Review by NathanGoodall2019 — COMMENTED** [review]
Just have a look at the comments I made but mainly I approve this.

File: futurenhs.automation/features/search.feature (hunk 1 above)
Comment (Alex-Grubba-CDS) [reply futurenhs.automation/features/search.feature:49]: Yes that's intentional, as the group _shouldn't_ exist in this search.

File: futurenhs.automation/features/userAccess.feature (hunk 3 above)
Comment (Alex-Grubba-CDS) [reply futurenhs.automation/features/userAccess.feature:5]: Mainly consistency and so it's easier to read. It's "cucumber formatter" in vscode.

File: futurenhs.automation/wdio.conf.js (hunk 4 above)
Comment (Alex-Grubba-CDS) [reply futurenhs.automation/wdio.conf.js:46]: Yes, currently they're organised in such a way for how data is manipulated as tests progress. However I want to try and do another review of all data and see about where this issue can be minimised even more so.

**Pull request merged**